.github/workflows/update-contribs.yml

```diff
@@ -0,0 +1,44 @@
+name: Update Contributors
+
+on:
+  pull_request:
```

Comment:
```suggestion
```

You'll want to remove this trigger before actually merging the PR, presumably

Reply:
oh yes - i just have this here so i can see how this runs when i update it. i do want to move this to a cron job once it works (and i'll rebase away all of my blblblblblb commits too 😆 i just haven't gotten this to actually create a pr yet is the problem.

Reply:
Seems like this wasn't actually removed before merging, so things might go haywire as soon as people push to other PRs.

Pro tip: The "Require conversations to be resolved before merging" branch protection rule is a good way to catch these sorts of things that can be easy to miss; we use it on a lot of our Python and Spyder repos.